Assets/Scripts/Controllers/Input Output/MouseController.cs

```diff
@@ -591,7 +591,7 @@ private void ShowFurnitureSpriteAtTile(string furnitureType, Tile t)
 
         Furniture proto = PrototypeManager.Furniture.Get(furnitureType);
 
-        go.transform.position = new Vector3(t.X + ((proto.Width - 1) / 2f), t.Y + ((proto.Height - 1) / 2f), WorldController.Instance.cameraController.CurrentLayer);
+        go.transform.position = new Vector3(t.X + ((sr.sprite.pivot.x / sr.sprite.pixelsPerUnit) - 0.5f), t.Y + ((sr.sprite.pivot.y / sr.sprite.pixelsPerUnit) - 0.5f), WorldController.Instance.cameraController.CurrentLayer);
```

Comment:
Fair point. I set up a system now that does that very thing. I set it up slightly different, using Vector3 addition, but it works still just as great!